style-dictionary/visual-refresh/colors.ts

```diff
@@ -261,6 +261,16 @@ const tokens: StyleDictionary.ColorsDictionary = {
   colorDropzoneBackgroundHover: { light: '{colorBlue200}', dark: '{colorBlue600}' },
   colorDropzoneTextActive: { light: '{colorGrey550}', dark: '{colorGrey800}' },
   colorDropzoneTextHover: { light: '{colorBlue800}', dark: '{colorWhite}' },
+  colorBackgroundSeverityCritical: { light: '{colorChartsRed1000}', dark: '{colorChartsRed300}' },
```

Comment:
Refer to the charts severity palette tokens when possible here.

Reply:
Updated in commit: https://github.com/cloudscape-design/components/pull/2592/commits/4794bea58d31ac6326a6a51bbdebbad7db9620cb